BEIMA.Backend/MongoService/MongoConnector.cs

```diff
@@ -417,5 +438,10 @@ public BsonDocument UpdateUser(BsonDocument doc)
         }
 
         #endregion
+
+        public FilterDefinition<BsonDocument> GetEqualsFilter(string key, dynamic value)
```

Comment:
Would this make more sense in a separate static filter generator class or something?

Reply:
Yeah this could probably be static.

Reply:
I implemented the MongoFilterGenerator static class and moved this method in there.